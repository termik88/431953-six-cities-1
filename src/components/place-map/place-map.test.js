import React from "react";
import renderer from "react-test-renderer";

import PlaceMap from "./place-map.jsx";

const mock = [
  {
    id: 0,
    premiumStatus: true,
    srcImage: ``,
    price: 120,
    rating: 93,
    title: `Beautiful &amp; luxurious apartment at great location`,
    type: `Apartment`,
    coordinates: [52.369553943508, 4.85309666406198]
  }
];

it(`PlacesList correctly renders`, () => {
  const tree = renderer.create(<PlaceMap
    offers = {mock}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
