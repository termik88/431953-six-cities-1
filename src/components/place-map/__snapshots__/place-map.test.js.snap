// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlacesList correctly renders 1`] = `
<section
  className="cities__map map"
  style={
    Object {
      "height": "100vh",
    }
  }
/>
`;
